container-storage-setup.1

```diff
@@ -33,6 +37,26 @@ command is not always sufficient to cleanup your
   container runtime environment. Other tools (\f[B]atomic storage reset\f[])
   use this command to cleanup all storage.
 
+\f[B]--version\f[]
+  Print version information
+
+.SH COMMANDS
+Following commands are supported.
+
+\f[B]create\f[] Create storage configuration
+
+\f[B]remove\f[]	Remove storage configuration
+
+\f[B]list\f[] List storage configuration
+
+\f[B]activate\f[] Activate storage configuration
+
+\f[B]deactivate\f[] Deactivate storage configuration
+
+\f[B]add-dev\f[] Add block device to storage configuration
+
+\f[B]export\f[] Export output file of storage configuration
```

Comment:
Export file which can be used to set environment variables for use by container runtimes?